LoRaEngine/LoraKeysManagerFacade/CreateEdgeDevice.cs

```diff
@@ -54,8 +49,9 @@ public CreateEdgeDevice(RegistryManager registryManager)
 
             // Get function facade key
             var base64Auth = Convert.ToBase64String(Encoding.Default.GetBytes($"{publishingUserName}:{publishingPassword}"));
-            var apiUrl = new Uri($"https://{Environment.GetEnvironmentVariable("WEBSITE_CONTENTSHARE")}.scm.azurewebsites.net/api");
-            var siteUrl = new Uri($"https://{Environment.GetEnvironmentVariable("WEBSITE_CONTENTSHARE")}.azurewebsites.net");
+            var apiUrl = new Uri($"https://{GetEnvironmentVariable("FACADE_HOST_NAME")}.scm.azurewebsites.net/api");
```

Comment:
is that wise to change the environment variable? Isn't this going to create a breaking change?

Reply:
I would also read it once into a variable and use the variable in the construction of the `Uri` instances so the environment variable name and reading doesn't have to be duplicated.

Reply:
I think we could do another PR dedicated to this topic. 

There is also a question around this: we use these urls to configure the deployment manifest, but we already have this information in the current request (hostname, code) since we are called by this information. 
This would simplify the configuration and the code. 

What do you think about it?